products/bash/release-products.sh

```diff
@@ -239,6 +240,56 @@ function release_apic {
 }
 
 
+# --- Asset Repo -------------------------------------------------------
+
+asset_repo_release_name=asset-repo-demo
+asset_repo_namespace=assetrepo
+asset_repo_chart=ibm-icp4i-asset-repo-prod
+asset_repo_pull_secret=ibm-entitlement-key
+asset_repo_block_storage_class=ibmc-block-gold
+asset_repo_file_storage_class=ibmc-file-gold
+
+asset_repo_values="\
+assetSync:
+  storageClassName: ${asset_repo_file_storage_class}
+couchdb:
+  persistentVolume:
+    storageClass: ${asset_repo_block_storage_class}
+global:
+  images:
+    pullSecret: ${asset_repo_pull_secret}
+    pullPolicy: Always
+license: accept
+selectedCluster:
+- ip: ${cp_console}
+  label: local-cluster
+  namespace: local-cluster
+  value: local-cluster
+"
+
+function release_asset_repo {
+    echo "Releasing Asset Repo..."
+    # Validate the environment
+    if ! kubectl get ns ${asset_repo_namespace} > /dev/null 2>&1; then
+      echo "There is no namespace '${asset_repo_namespace}'" 1>&2
+      exit 1
+    else
+      echo "Namespace '${asset_repo_namespace}' exists"
+    fi
+    if ! kubectl get secret ${asset_repo_pull_secret} -n ${asset_repo_namespace} > /dev/null 2>&1; then
+      echo "There is no '${asset_repo_pull_secret}' in namespace '${asset_repo_namespace}'" 1>&2
+      exit 1
+    else
+      echo "The secret '${asset_repo_pull_secret}' exists in the namespace '${asset_repo_namespace}'"
+    fi
+    # Create the Helm values file
+    echo "${asset_repo_values}" > asset-repo-values.yaml
+    echo "Asset repository values.yaml created"
+    # Create the release
+    echo "Creating the asset repo release"
+    helm install ${chart_repo}/${asset_repo_chart} --name ${asset_repo_release_name} --namespace ${asset_repo_namespace} --values asset-repo-values.yaml --tls --debug
```

Comment:
And the `--debug` here should be removed.

Reply:
Done